src/lib.rs

```diff
@@ -97,3 +96,40 @@ fn dotenv_files(matches: clap::ArgMatches) -> Result<Vec<PathBuf>, Error> {
 
     Ok(paths)
 }
+
+pub fn extract_key(input: &str) -> String {
```

Comment:
I think this function should be implemented for the struct `LineEntry` and should return the type [Option](https://doc.rust-lang.org/std/option):
```rust
impl LineEntry {
    fn extract_key() -> Option<String>
}
```